pyhoma/models.py

```diff
@@ -153,6 +153,11 @@ def __contains__(self, name: str) -> bool:
     def __getitem__(self, name: str) -> Optional[State]:
         return next((state for state in self._states if state.name == name), None)
 
+    def __setitem__(self, name: str, state: State) -> None:
```

Comment:
What is the use-case for this setter? And it can not be used to overwrite an existing state?

Reply:
it allows to dynamically add a state if it was not present at the entity's initialization which happens for the Smart Thermostat.